boot/zephyr/prj.conf

```diff
@@ -6,7 +6,7 @@ CONFIG_SYS_POWER_MANAGEMENT=n
 CONFIG_MAIN_STACK_SIZE=10240
 CONFIG_MBEDTLS_CFG_FILE="mcuboot-mbedtls-cfg.h"
 
-CONFIG_BOOT_ENCRYPT_RSA=n
+CONFIG_BOOT_ENCRYPT_RSA=y
```

Comment:
Why are we changing the default for the encryption feature? Shouldn't that still be up to the user?

Reply:
You're right. It's a testing leftover of my part.